CanvasPlusPlayground/Views/AggregatedAssignmentsView.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  AggregatedAssignmentsView.swift
+//  CanvasPlusPlayground
+//
+//  Created by Ethan Fox on 10/18/24.
+//
+
+import SwiftUI
+
+struct AggregatedAssignmentsView: View {
+    @Environment(CourseManager.self) var courseManager
+    @State var course_assignments: [(Assignment, Course)] = []
```

Comment:
nit: naming convention
```suggestion
    @State var courseAssignments: [(Assignment, Course)] = []
```